src/pages/CreatorCenter/components/ClassRegist/components/InputForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../../../../../styles/theme';
+
+function InputForm({ type, subject, description, borderUnUse }) {
+  return (
+    <Container borderUnUse={borderUnUse}>
+      <Subject>{subject}</Subject>
+
+      {type === 'list' ? (
+        <Select name="levelList">
+          <option value="">선택안함</option>
+          <option value="입문자">입문자</option>
+          <option value="초급자">초급자</option>
+          <option value="중급자">중급자</option>
+          <option value="준전문가">준전문가</option>
+          <option value="전문가">전문가</option>
+        </Select>
+      ) : type === 'textarea' ? (
+        <Textarea />
+      ) : (
+        <Input type={type} placeholder={description} />
+      )}
+    </Container>
+  );
+}
+const Container = styled.div`
+  ${props =>
+    props.borderUnUse === true ? null : `border-bottom: 1px solid #e5e5e5;`}
+  padding: 15px;
+`;
```

Comment:
```suggestion
const Container = styled.div`
  ${props => (props.borderUnUse ? null : `border-bottom: 1px solid #e5e5e5;`)}
  padding: 15px;
`;
```
이미 props로 받는 값이 boolean 값이기 때문에 위와 같이 수정하는 방법으로 구현해주세요!